app/frontend/vite.config.ts

```diff
@@ -14,5 +14,13 @@ export default defineConfig({
         alias: {
             "@": path.resolve(__dirname, "./src")
         }
+    },
```

Comment:
Is this change supposed to be in the PR as well?

Reply:
Noo sorry reverting